concepticondata/conceptrelations.tsv

```diff
@@ -530,3 +529,80 @@ SOURCE	SOURCE_GLOSS	RELATION	TARGET	TARGET_GLOSS
+3186	BE DEAD OR DEATH OR DIE	narrower	1494	DIE
+3186	BE DEAD OR DEATH OR DIE	narrower	1625	DEAD
+3186	BE DEAD OR DEATH OR DIE	narrower	910	DEATH
+3187	DROWNED	resultsin	1904	DROWN
+3188	MIGHTY OR POWERFUL OR STRONG	narrower	785	STRONG
+3188	MIGHTY OR POWERFUL OR STRONG	narrower	3001	POWERFUL
+680	BREAK (DESTROY OR GET DESTROYED)	narrower	3020	BREAK (BREAKING)
+2558	BREAK (CLEAVE)	transitiveof	3020	BREAK (BREAKING)
+3192	BRONZE OR COPPER	narrower	2313	BRONZE
+3192	BRONZE OR COPPER	narrower	1825	COPPER
+3194	PAINT (WALL) OR PAINTING	narrower	3193	PAINT (WALL)
+3194	PAINT (WALL) OR PAINTING	narrower	2001	PAINTING
+692	BRING	narrower	700	CARRY
+1494	DIE	narrower	2865	DIE (FROM ACCIDENT)
+3195	DAMAGE OR INJURE	narrower	1116	DAMAGE
+3195	DAMAGE OR INJURE	narrower	44	INJURE
+987	REMAIN	narrower	3196	BE LEFT
+3198	NOTHING OR ZERO	narrower	687	NOTHING
+3198	NOTHING OR ZERO	narrower	1294	ZERO
+2896	CAUSE TO (LET)	narrower	1003	PERMIT
+3201	HINDER OR PREVENT	narrower	554	HINDER
+3201	HINDER OR PREVENT	narrower	1550	PREVENT
+3201	HINDER OR PREVENT	narrower	92	SPOIL (SOMEBODY OR SOMETHING)
+3203	CANNON OR GUN	narrower	3202	CANNON
+3203	CANNON OR GUN	narrower	1566	GUN
+1945	IDOL	narrower	3205	GRAVEN IMAGE
+2138	LEGENDARY CREATURE	narrower	1138	FAIRY
+3206	ELF OR FAIRY	narrower	1138	FAIRY
+3206	ELF OR FAIRY	narrower	1037	ELF
+1352	KNIFE	narrower	3210	KNIFE (FOR EATING)
+3207	GO UP OR RISE (ONESELF)	narrower	1512	RISE (ONESELF)
+3207	GO UP OR RISE (ONESELF)	narrower	705	GO UP (ASCEND)
+3208	CENTER OR MIDDLE	narrower	491	CENTER
+3208	CENTER OR MIDDLE	narrower	1093	MIDDLE
+1092	HOPE	narrower	3211	HOPE (FAITH)
+1092	HOPE	narrower	3212	HOPE (SOMETHING)
+3213	HIRE OR RENT	narrower	856	HIRE
+3214	CRAWL OR CREEP	narrower	96	CRAWL
+3214	CRAWL OR CREEP	narrower	3215	CREEP
+3216	DRIP OR DROP SOMETHING	narrower	808	DRIP (FALL IN GLOBULES)
+3216	DRIP OR DROP SOMETHING	narrower	2866	DROP (SOMETHING)
+3217	TIN OR TINPLATE	narrower	893	TIN (METAL)
+3217	TIN OR TINPLATE	narrower	358	TINPLATE
+3219	SPLASH OR SPRAY	narrower	3218	SPRAY
+3219	SPLASH OR SPRAY	narrower	365	SPLASH
+3220	MATERNAL UNCLE	narrower	2694	MATERNAL UNCLE (HUSBAND OF ONE'S MATERNAL AUNT)
+3220	MATERNAL UNCLE	narrower	2692	MATERNAL UNCLE (MOTHER'S BROTHER)
+926	RICE	narrower	1191	COOKED RICE
+2641	MYTHICAL BEING	sameas	2138	LEGENDARY CREATURE
+3223	BRANCH OR TWIG	narrower	1531	BRANCH
+3223	BRANCH OR TWIG	narrower	3222	TWIG
+3225	SQUEEZE OR WRING	narrower	3224	WRING
+3225	SQUEEZE OR WRING	narrower	1414	SQUEEZE
+3226	FASTEN OR SQUEEZE	narrower	1414	SQUEEZE
+3226	FASTEN OR SQUEEZE	narrower	1094	FASTEN
+1587	SMELL (STINK)	narrower	3229	STINK
+3228	BALL OR SPHERE	narrower	3227	SPHERE
+3228	BALL OR SPHERE	narrower	1845	BALL
+1088	SINK (DESCEND)	narrower	2908	SINK (DISAPPEAR IN WATER)
+3231	DEITY	narrower	1944	GOD
+588	BROKEN	narrower	3190	BROKEN (NOT WORKING)
+588	BROKEN	narrower	3191	BROKEN (SHATTERED)
+1352	KNIFE	narrower	3233	KITCHEN KNIFE
+580	FUR	narrower	2678	HAIR OR FUR
```

Comment:
Important: this has been swapped:

HAIR OR FUR "the narrower concept is" FUR

right?

Reply:
Actually not. FUR refers to body hair of mammals or to a pelt made of it, whereas HAIR OR FUR only refers to body hair of mammals. Not optimal, but that's what is/was linked to them.

Reply:
The definitions are rather overlapping instead of proper hyperonyms.
And with the one "HAIR OR FUR"-list we don't even know what it refers to exactly.
We do have 3 for "PELT", though.

Reply:
I agree with this, but I can see other cases of overlapping for South American languages (but I cannot find any actual wordlist). In Southern Brazilian Portuguese we have "pelo" which refers to body hair of animals and humans (but not head hair), "pelagem" which is only FUR, "pele/pelego" which is both FUR and PELT (while "pele" is also SKIN).

Once more, the issue is to know that Concepticon does not map reality into words, but establishes links among concept lists. We should only list disjunctive CONCEPT if such disjunction differentiates between at least two concept lists that we included. If new concept lists will demand more work to fix it, than we will work on it -- otherwise, we will fail as every single attempt at an universal language.